documentation/modules/auxiliary/multidrop.md

```diff
@@ -0,0 +1,92 @@
+This module dependent on the given filename extension creates either a .lnk, .scf, .url, desktop.ini file which includes a reference to 
+the the specified remote host, causing SMB connections to be initiated from any user that views the file. This allows for NetNTLM hashes to be captured
+by a listening user.
+
+## Vulnerable Application
+
+Microsoft Windows
+
+## Verification Steps
+
+  1. Install the application
+  2. Start msfconsole
+  3. Do: ```use auxiliary/multidrop```
+  4. Customise Options as required
+  5. Do: ```run```
+  6. A file pointing back to the listening host will then be generated.
+  7. Configure auxiliary/server/capture/smb or similar to capture hashes.
+  8. Upload the document to an open share or similar and wait for hashes.
+
+## Options
+
+FILENAME - This option allows you to customise the generated filename and filetpye that is generated.
+This can be changed using set FILENAME
+
+To generate desktop.ini configure a filename of desktop.ini
+To generate a scf file configure a filename of anyname.scf
+To generate a url file configure a filename of anyname.url
+To generate a lnk file configure a filename of anyname.lnk
+
+File type generation is based on the file extension.
+
+LHOST - This option allows you to set the IP address of the SMB Listener that the document points to
+This can be changed using set LHOST 192.168.1.25
+
+## Scenarios
+
+### Version of software and OS as applicable

```

Comment:
This header is meant to be replaced with the version of software and OS as applicable. ie, `### Microsoft Windows`

Reply:
fixed